capsul/engine/database_json.py

```diff
@@ -70,7 +71,10 @@ def set_path_metadata(self, path, metadata, named_directory=None):
         path = metadata['path']
         named_directory = metadata['named_directory']
         
-        self.json_dict.setdefault('path_metadata', {})[(named_directory, path)] = doc
```

Comment:
@sapetnioc I tried to fix this incorrect code, but I am not sure I understand what it is supposed to do. Please check if my fix is correct.

Reply:
That was ok. There was another mistake just above that I fixed.